src/libpgagroal/management.c

```diff
@@ -78,12 +73,12 @@ pgagroal_management_request_flush(SSL* ssl, int socket, int32_t mode, char* data
    struct json* j = NULL;
    struct json* request = NULL;
 
-   if (create_header(MANAGEMENT_FLUSH, compression, encryption, output_format, &j))
+   if (pgagroal_management_create_header(MANAGEMENT_FLUSH, compression, encryption, output_format, &j))
```

Comment:
Is this related to this patch? Should be into another PR?
Also true for the following ones that are not related to user management,.